Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
The most portable shebang: https://stackoverflow.com/a/10383546/2231969

Reply:
yep, but for this script we don't want it to be portable ! it is only for ubuntu and a recently debian so /bin/bash is good !